server/controllers/finance/journal/index.js

```diff
@@ -119,11 +120,6 @@ function getTransaction (req, res, next){
  * POST /journal/:uuid/reverse
  */
 function reverse(req, res, next) {
-<<<<<<< 3fa08c9a78e043908319ddaad1dc050257f02b0d
-=======
-  const uid = db.bid(req.params.uuid);
-  const userId = req.session.user.id;
->>>>>>> feat(entities): introduce entity_map
```

Comment:
Ah, I didn't realize this was from such an old commit.  That's what happens when you rebase your branch too many times ...